canonical-web-and-design/jaas-dashboard
# Add a confirmation dialog to config action buttons.

## Done

Add a confirmation dialog to the configuration action buttons [cancel, save]

Note: use w=1 for review.

## QA

Use the demo link and the model `charmed-kubernetes` with the app `containerd` then branch out to try other models/apps to chaos monkey it.
- Without editing any config, clicking the `X` or `Cancel` should just close the panel.
- If you edit a config and click `Cancel` it should confirm, try both buttons, Also do the same with the `X`
- Now edit a config and click save, it should confirm, try both buttons.
- Chaos monkey!

## Details

Fixes #850 
Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1581

## Screenshots

![Screen Shot 2021-02-02 at 3 39 13 PM](https://user-images.githubusercontent.com/532033/106666335-1601f600-656d-11eb-8096-7f48fa5c9a1a.png)
![Screen Shot 2021-02-02 at 3 39 24 PM](https://user-images.githubusercontent.com/532033/106666337-169a8c80-656d-11eb-830d-7e5b18d706e4.png)
